modules/apps/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/service/impl/MBMessageLocalServiceImpl.java

```diff
@@ -2046,6 +2045,32 @@ private String _getLocalizedRootCategoryName(Group group, Locale locale) {
 		}
 	}
 
+	private String _getMessageSiblings(
```

Comment:
Move this to the util class and handle all the details there. MBMessageLocalService shouldn't have to worry how these messages are composed.